components/FavouriteList.tsx

```diff
@@ -0,0 +1,37 @@
+import {FlatList, TouchableOpacity, View} from "react-native";
+import MuseumTile from "./MuseumTile";
+import {useEffect, useState} from "react";
+import {getFavourites, toggle} from "../async_storage/LocalStorage";
+import {useIsFocused} from "@react-navigation/native";
+
+
+export default function FavouriteList() {
+    const [favIds, setFavIds] = useState<Array<number>>()
+
+    useEffect(() => {
+        getFavourites().then(setFavIds)
+    }, [useIsFocused()]);
+
+
+    function onTileTap(value: number) {
+        toggle(value).then((ids) => setFavIds(ids))
+    }
+
+    const renderItem = ({item}: { item: number }) => {
+
+        return (
+            <TouchableOpacity>
+                <MuseumTile id={item} onTap={() => onTileTap(item)} selected={favIds?.includes(item) ?? false}/>
```

Comment:
I think we can simplify this by passing an empty array as initial prop to `favIds` :)

```suggestion
                <MuseumTile id={item} onTap={() => onTileTap(item)} selected={favIds.includes(item)}/>
```